pkg/config/config.go

```diff
@@ -32,6 +32,9 @@ const (
 
 	// DefaultKubeConformanceImageURL is the URL of the docker image to run for the kube conformance tests.
 	DefaultKubeConformanceImageURL = "gcr.io/heptio-images/kube-conformance"
+	// UpstreamKubeConformanceImageURL is the URL of the docker image to run for
+	// the kube conformance tests which is maintained by upstream Kubernetes.
+	UpstreamKubeConformanceImageURL = "gcr.io/google-containers/conformance"
 	// DefaultKubeConformanceImageTag is the default tag of the conformance image
 	DefaultKubeConformanceImageTag = "latest"
```

Comment:
I don't think we should ever run 'latest` on an image. If someone makes a change upstream and breaks the tests, users will get false positives.

Reply:
Yea, latest probably isn't the smartest default. Should we just put that at the latest tag as of a given release and then each time we release Sonobuoy we update it? We'll just need to add that to the release notes (If you agree I'll make an issue for that too)

Reply:
So I'm so glad you brought up the questions w.r.t. `latest`.

TIL: Sonobuoy gen defaults to using `latest` and sonobuoy run defaults to `auto`. I'm assuming the reason is that `auto` normally gets resolved at a later time in the sonobuoy run logic and so `gen` wanted a sane default that could work -> latest.

I'll look into it but based on that understanding I think it makes sense to make a new issue to change/track that. An ideal solution in my mind looks like changing the logic of `gen` so that it can have the same resolution logic as `run` and only if it can't reach out to the server does it default to _something_. In which case we could specify a default like `latest` or just another static tag like `1.13.0` or whatever is the most recent release when we build/release sonobuoy.

Reply:
https://github.com/heptio/sonobuoy/issues/619

Reply:
My question is then why does sonobuoy gen and sonobuoy run use different logic? They should follow the same logic path. That could be the core issue here.

Reply:
Agree; until this I didnt realize they differed here. I was confused why `gen` was using `latest` when I thought the default was `auto`